flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -265,30 +268,35 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
     }
 
     await _hub.configureScope((scope) {
-      scope.span ??= _transaction;
+      scope.span = _transaction;
```

Comment:
i guess this was done to preserve a transaction started by the user and bound to the scope
```suggestion
      scope.span ??= _transaction;
```